common/tests/fixtures/cardano/sign_tx.show_details.json

```diff
@@ -4,54 +4,6 @@
     "passphrase": ""
   },
   "tests": [
-    {
-      "description": "Mainnet transaction without change",
-      "parameters": {
-        "protocol_magic": 764824073,
-        "network_id": 1,
-        "fee": 42,
-        "ttl": 10,
-        "validity_interval_start": null,
-        "certificates": [],
-        "withdrawals": [],
-        "auxiliary_data": null,
-        "inputs": [
-          {
-            "path": "m/44'/1815'/0'/0/1",
-            "prev_hash": "1af8fa0b754ff99253d983894e63a2b09cbb56c833ba18c3384210163f63dcfc",
-            "prev_index": 0
-          }
-        ],
-        "outputs": [
-          {
-            "address": "Ae2tdPwUPEZCanmBz5g2GEwFqKTKpNJcGYPKfDxoNeKZ8bRHr8366kseiK2",
-            "amount": "3003112"
-          }
-        ],
-        "mint": [],
-        "script_data_hash": null,
-        "collateral_inputs": [],
-        "required_signers": [],
-        "collateral_return": null,
-        "total_collateral": null,
-        "reference_inputs": [],
-        "signing_mode": "ORDINARY_TRANSACTION",
-        "additional_witness_requests": [],
-        "include_network_id": false,
-        "tag_cbor_sets": false
-      },
-      "result": {
-        "tx_hash": "73e09bdebf98a9e0f17f86a2d11e0f14f4f8dae77cdf26ff1678e821f20c8db6",
-        "witnesses": [
-          {
-            "type": 0,
-            "pub_key": "89053545a6c254b0d9b1464e48d2b5fcf91d4e25c128afb1fcfc61d0843338ea",
-            "signature": "da07ac5246e3f20ebd1276476a4ae34a019dd4b264ffc22eea3c28cb0f1a6bb1c7764adeecf56bcb0bc6196fd1dbe080f3a7ef5b49f56980fe5b2881a4fdfa00",
-            "chain_code": "26308151516f3b0e02bb1638142747863c520273ce9bd3e5cd91e1d46fe2a635"
-          }
-        ]
-      }
-    },
```

Comment:
I understand you're removing the tests because they produce the same series of screens, but aren't they still valuable for testing the transaction serialization and signing code?

Edit: aren't, because the same transactions exist in `cardano/sign_tx.json`.